CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+# Contributing
+
+This project welcomes contributions and suggestions. Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.opensource.microsoft.com.
+
+When you submit a pull request, a CLA bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., status check, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
+
+This project has adopted the Microsoft Open Source Code of Conduct. For more information see the Code of Conduct FAQ or contact opencode@microsoft.com with any additional questions or comments.
+
+<br>
+
+## Contributer Getting Started
+
+### SQL Setup
+
+This requires already having a SQL database. If you need to create a SQL database, we recommend one of 2 options:
+- [Local SQL Server running in a Docker container](https://docs.microsoft.com/sql/linux/quickstart-install-connect-docker)
+- [Azure SQL Database](#Create-Azure-SQL-Database).
+
+A primary key must be set in your SQL table before using the bindings. To do this, run the below SQL commands in the SQL query editor.
+
+1. Ensure there are no NULL values in the primary key column. The primary key will usually be an ID column.
+
+    ```sql
+    ALTER TABLE ['your table name'] alter column ['column to be primary key'] int NOT NULL
+    ```
+
+2. Set primary key column.
+
+    ```sql
+    ALTER TABLE ['your table name'] ADD CONSTRAINT PKey PRIMARY KEY CLUSTERED (['column to be primary key']);
+    ```
+
+3. Congrats on setting up your database! Now continue to set up your local environment and complete the quick start.
+
+### Set Up Development Environment
+
+1. [Install VS Code](https://code.visualstudio.com/Download)
+   
+2. Clone repo and open in VS Code:
+
+```bash
+git clone https://github.com/Azure/azure-functions-sql-extension
+cd azure-functions-sql-extension
+code .
+```
+3. Install extensions when prompted after VS Code opens
+   - Note: This includes the Azure Functions, C#, and editorconfig extensions
+
+4. Get your SqlConnectionString. Your connection string can be found in your SQL database resource by going to the left blade and clicking 'Connection strings'. Copy the Connection String.
```

Comment:
This is only for Azure, it's probably fine just to skip this (or point to some generic documentation on connection strings)